subscriptions/tasks.py

```diff
@@ -327,23 +378,15 @@ def run(self, subscription_id, **kwargs):
         try:
             subscription = Subscription.objects.get(id=subscription_id)
             if subscription.process_status == 0:
-                scheduler = self.scheduler_client()
-                # get the messageset length for frequency calc
-                message_count = subscription.messageset.messages.filter(
-                    lang=subscription.lang).count()
-                # next_sequence_number is for setting non-one start position
-                # frequency is the number of messages that should be sent
-                frequency = (
-                    message_count - subscription.next_sequence_number) + 1
-                # Build the schedule POST create object
                 schedule = {
-                    "frequency": frequency,
+                    "frequency": None,
```

Comment:
The other issue is with the calculating setup, if something goes wrong with one message send and it doesn't update the next_sequence_number the way it should, the scheduler would have moved on, but not the message sequence number. Which would also mean you don't hit the end of the message set.